test/endtoend/wdio.remote.conf.js

```diff
@@ -20,8 +20,8 @@ exports.config = Object.assign(base.config, {
     { browserName: 'MicrosoftEdge', version: '14', platform: 'Windows 10' },
     { browserName: 'MicrosoftEdge', version: '17', platform: 'Windows 10' },
     // { browserName: 'safari', version: '11.0', platform: 'macOS 10.12' },
-    { browserName: 'safari', version: '10.0', platformName: 'iOS', platformVersion: '10.0', deviceOrientation: 'portrait', deviceName: 'iPhone 6 Simulator', appiumVersion: '1.7.1' },
-    { browserName: 'chrome', platformName: 'Android', platformVersion: '6.0', deviceOrientation: 'portrait', deviceName: 'Android Emulator', appiumVersion: '1.7.1' }
+    { browserName: 'safari', version: '10.0', platformName: 'iOS', platformVersion: '12.2', deviceOrientation: 'portrait', deviceName: 'iPhone 6 Simulator', recordScreenshots: false },
+    { browserName: 'chrome', platformName: 'Android', platformVersion: '6.0', deviceOrientation: 'portrait', deviceName: 'Android Emulator' }
```

Comment:
So iOS version `12.2`, but Safari still `10.0`? 🤔 

This is the config I get from the [Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/):

```
caps['browserName'] = 'Safari';
caps['deviceName'] = 'iPhone 6 Simulator';
caps['deviceOrientation'] = 'portrait';
caps['platformVersion'] = '12.2';
caps['platformName'] = 'iOS';
```

Also, why `recordScreenshots: false`, cause we have video?

Reply:
ahh sick i didn't know about the configurator

recordScreenshots was a stab in the dark because the test timed out for no apparent reason and i saw someone somewhere™ say that happened for them with screenshots, but it didn't fix it here